weather_man.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import os
+
+from data_reader import DataReader
+
+
+weather_report = {}
+data_reader = DataReader()
+
+
+def is_dir(dirname):
+	if not os.path.isdir(dirname):
+		error_message = '{0} is not a directory'.format(dirname)
+		raise argparse.ArgumentTypeError(error_message)
+
+	else:
+		return dirname
+
+def get_weather_data(arguments):
+
+	if arguments.report_type == 1:
+		annual_report = data_reader.generate_annual_report(arguments.data_dir)
+		if annual_report:
+			data_reader.print_annual_report(annual_report)
+
+	elif arguments.report_type == 2:
+		annual_report = data_reader.generate_annual_report(arguments.data_dir)
```

Comment:
same in above conditional statement so we could write it out of condition block.